modules/change-collection-method.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-operator.adoc
+:_mod-docs-content-type: CONCEPT
+[id="change-collection-method_{context}"]
+= Changing the collection method to EBPF or CORE_BPF
+
+[role="_abstract"]
+If the cluster that you are upgrading contains the `SecuredCluster` CR, you must ensure that the per node collection setting is set to `CORE_BPF` before you upgrade, if you are upgrading from 4.1 or later. Otherwise, set the collection method to `EBPF`. To set the collection method to `EBPF`, you must set the `forceCollection` option to `true` after the upgrade and make sure that the collection method is `EBPF`.
+
+.Procedure
+
+. In the {ocp} web console, go to the {product-title-short} Operator page.
+. In the top navigation menu, select *Secured Cluster*.
+. Click the instance name, for example, *stackrox-secured-cluster-services*.
+. Use one of the following methods:
+* In the *Form view*, under *Per Node Settings* -> *Collector Settings* -> *Collection*, select *CORE_BPF*.
+* Click *YAML* to open the YAML editor and locate the `spec.perNode.collector.collection` attribute. If the value is `KernelModule`, then change it to `CORE_BPF`.
+. Click *Save.*
```

Comment:
There should probably be instructions on how to set forceCollection. I suggest the following
```
To set forceCollection to true  Click *YAML* to open the YAML editor and locate the `spec.perNode.collector.forceCollection` attribute and set it to true.
```

Reply:
Added a new section here for this.